src/configuration/services/saas.md

```diff
@@ -0,0 +1,60 @@
+
+Magento allows you to link your Magento store to a SaaS service through the Admin UI. By configuring and managing SaaS Environments, you can connect to services such as Product Recommendations.
+
+When you create a [Magento account]({% link magento/magento-account.md %}), identified by a MageID, you can generate a Magento API key to connect to the SaaS services within Magento. To connect to Magento Commerce SaaS services, the API key needs to be generated by the license-holder. Then the API key can be passed to the systems integrator or development team, who can then manage the SaaS projects and environments on behalf of the license-holder.
+
+Each MageID can have one SaaS project and within that SaaS project you can have one or more SaaS environments depending on whether or not you have a license:
+
+-  **Magento Commerce** - One production environment; two testing environments
+
+-  **Magento Open Source** - One production environment; no testing environments
+
+{:.bs-callout-info}
+SaaS projects and environments are separate but analogous concepts to Magento Cloud projects and environments.
+
+## API keys {#apikey}
+
+![]({% link images/images/api-key-saas.png %}){: .zoom}
+_API Key_
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Production Api Key|Global|API key that identifies your production environment.|
+|Sandbox Api Key|Global|API key that identifies your testing environment.|
+
+## SaaS Environments {#envid}
+
+![]({% link images/images/saas-env-saas.png %}){: .zoom}
+_SaaS Environment_
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|SaaS Project|Global|Name of the SaaS project that groups all of your SaaS environments.|
+|SaaS Environment|Global|Lists of environment IDs and names in the specified SaaS Project.|
+
+## Create SaaS Environment
+
+To create a SaaS Environment, request the Magento API key from the Magento license holder for your store.
+
+1. In the Magento Admin, choose **Stores** > Settings > **Configuration** > **Services** > **Magento Services**.
+
+1. In the **API Keys** section, paste your key value into the **Production Api Key** field regardless of whether you are in a production environment or non-production environment.
+
+1. Click **Save Config**.
+
+   When you save, if there are any SaaS projects associated with your API key, those projects will appear in the the **SaaS Project** field located in the **SaaS Environment** section.
+
+1. If there are not any SaaS projects created, click the **Request New Project** to display the **Project Name** field and enter a name for your SaaS project.
+
+   Magento creates a SaaS project and as many SaaS environments you are entitled to.
+
+1. Select the **SaaS Environment** to use for the current configuration of your Magento store.
+
+To change the SaaS project or environment names, click the **Rename this Project** or **Rename this Environment** respectively.
```

Comment:
Do we need to Save Config here?

Reply:
no; the page refreshes automatically.

Reply:
Usually in the Configuration in the Admin you need to click the Save Config button when you make a change to the settings. If not, then fine.

Reply:
yep, usually you do, but not in this case